tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -87,3 +87,10 @@ Feature: An administrator wants to import a user using the commandline
     When a user is imported from path "simpleExport/unknown" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "user.json not found"
+
```

Comment:
I think we can add  at top
```
        As a user
        I want to ......
        So that ..........
 ```

Reply:
let's do this as a part of this issue https://github.com/owncloud/data_exporter/issues/226 and not in this PR